agent_api_rest/src/identity/profiles/mod.rs

```diff
@@ -0,0 +1,78 @@
+use crate::error::IntoApiErrorExt;
+use crate::handlers::{command_handler, query_handler};
+use crate::utils::serde_explicit_null;
+use agent_identity::profile::aggregate::Source;
+use agent_identity::profile::command::ProfileCommand;
+use agent_identity::state::IdentityState;
+use agent_identity::state::{query_profile, PROFILE_ID};
+use agent_shared::config::Logo;
+use axum::{
+    extract::State,
+    response::{IntoResponse, Response},
+    Json,
+};
+use http_api_problem::ApiError;
+use hyper::StatusCode;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Should we `#[serde(deny_unknown_fields)]` to throw an error when unknown keys are supplied during PATCH? Do you think this adds more explicitness or is it not necessary?
Reason I am asking this, is because I am currently getting a clean `200 OK` with no further response body when supplying `display_name` instead of `displayName`, so I thought the update was successful.